backend/src/models/UserModel.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+var __extends = (this && this.__extends) || (function () {
+    var extendStatics = Object.setPrototypeOf ||
+        ({ __proto__: [] } instanceof Array && function (d, b) { d.__proto__ = b; }) ||
+        function (d, b) { for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p]; };
+    return function (d, b) {
+        extendStatics(d, b);
+        function __() { this.constructor = d; }
+        d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
```

Comment:
Please let's try to avoid single letter variables, is very difficult to understand the code in this way
why do we need to use __proto__ ?

Reply:
This is re utilized from react-backend repository which was wrote like this, I can do a deep research to refactor the backend and be more descriptive from a intent perspective